tests/nightly/model_backwards_compatibility_check/train_mxnet_legacy_models.sh

```diff
@@ -57,10 +57,10 @@ do
 	fi
 
 	## If MXNet major version starts with a number >=1. with a wildcard match for the minor version numbers
-	if [[ $version = [1-9]* ]]
+	if [[ $version = [1-9].[0-9].[0-9] ]]
```

Comment:
I don't think this will handle multi digit versions.

Reply:
Good point. Done 16d320a926d4f23265f730d2fc197ecec5bc4a2e